src/main/java/frc/robot/Robot.java

```diff
@@ -19,9 +23,22 @@
  * project.
  */
 public class Robot extends TimedRobot {
-  
+     
```

Comment:
this code should not be in robot, you need to create a subsystem object and put it there. use last year's code (Robot 2020) for example subsystems.